src/main/scala/ch/epfl/scala/sbtmissinglink/MissingLinkPlugin.scala

```diff
@@ -31,16 +32,25 @@ object MissingLinkPlugin extends AutoPlugin {
   override def trigger: PluginTrigger = allRequirements
 
   val configSettings: Seq[Setting[_]] = Def.settings(
-    missinglinkCheck := {
+    missinglinkExcludeDependencies := Nil,
+    missinglinkConflicts := {
       val log = streams.value.log
-
-      val cp = Attributed.data(fullClasspath.value)
+      val cp = fullClasspath.value
       val classDir = (classDirectory in Compile).value
-      val conflicts = loadArtifactsAndCheckConflicts(cp, classDir, log)
-
+      loadArtifactsAndCheckConflicts(cp, missinglinkExcludeDependencies.value, classDir, log)
+    },
+    missinglinkIgnoreConflicts := Nil,
+    missinglinkCheck := {
+      val log = streams.value.log
+      val allConflicts = missinglinkConflicts.value
+      val ignores = missinglinkIgnoreConflicts.value.map(_.unanchored)
+      val conflicts = allConflicts.filterNot { c =>
+        val descriptions = ConflictDescription(c).allDescriptions
+        ignores.exists(r => descriptions.exists(d => r.findFirstIn(d).nonEmpty))
```

Comment:
This text search in the descriptions of the conflicts seems very brittle to me. It also does not appear to match how the original Maven plugin specifies conflicts; the latter looks more robust and safe.

Consider following the strategy used by the original Maven plugin for the configuration and execution of conflicts. See https://github.com/spotify/missinglink/blob/d2d25cd15733c232ba6bea59d1fea274c2a13346/maven-plugin/src/main/java/com/spotify/missinglink/maven/CheckMojo.java#L217

Reply:
it is a bit brittle, but i wanted to have an easier api - you can filter out exactly what you see in log
the extraction of `missinglinkConflicts` was done specifically so that you could have a less brittle (but much more involved) way